### ISSUE TITLE
UnicodeEncodeError: 'ascii' codec can't encode characters

### ISSUE THREAD

**Issue by kazhuravlev** [bogdal/sentry-youtrack#12]
On create new issue: 

UnicodeEncodeError: 'ascii' codec can't encode characters in position 0-5: ordinal not in range(128)

```
UnicodeEncodeError: 'ascii' codec can't encode characters in position 0-5: ordinal not in range(128)
  File "django/core/handlers/base.py", line 112, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "django/utils/decorators.py", line 29, in _wrapper
    return bound_func(*args, **kwargs)
  File "django/utils/decorators.py", line 99, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "django/utils/decorators.py", line 25, in bound_func
    return func(self, *args2, **kwargs2)
  File "sentry/web/frontend/base.py", line 165, in dispatch
    return self.handle(request, *args, **kwargs)
  File "sentry/web/frontend/group_plugin_action.py", line 25, in handle
    response = plugin.get_view_response(request, group)
  File "sentry_youtrack/plugin.py", line 113, in get_view_response
    return super(YouTrackPlugin, self).get_view_response(request, group)
  File "sentry/plugins/base/v1.py", line 296, in get_view_response
    response = self.view(request, group)
  File "sentry_youtrack/plugin.py", line 131, in view
    return view(request, group, **kwargs)
  File "sentry/plugins/bases/issue.py", line 163, in view
    form = self.get_new_issue_form(request, group, event)
  File "sentry_youtrack/plugin.py", line 79, in get_new_issue_form
    initial=self.get_initial_form_data(request, group, event))
  File "sentry_youtrack/forms.py", line 31, in __init__
    self.add_project_fields(project_fields)
  File "sentry_youtrack/forms.py", line 36, in add_project_fields
    form_field = self._get_form_field(field)
  File "sentry_youtrack/forms.py", line 68, in _get_form_field
    'initial': self._get_initial(project_field['name'])}
  File "sentry_youtrack/forms.py", line 57, in _get_initial
    field_key = md5(field_name).hexdigest()
```

**Comment by bogdal** [comment]
@kazhuravlev Which version of `sentry` do you use?

**Comment by homberger** [comment]
We are also affected by this error.

Sentry version: 8.5.1
sentry-youtrack plugin version: 0.3.1
YouTrack version: YouTrack 6.5 Build 17057

```
[ERROR] Internal Server Error: /domain/project/issues/1670/actions/youtrack/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/sentry/../django/core/handlers/base.py", line 112, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python2.7/site-packages/sentry/../django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/sentry/../django/utils/decorators.py", line 29, in _wrapper
    return bound_func(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/sentry/../django/utils/decorators.py", line 99, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/sentry/../django/utils/decorators.py", line 25, in bound_func
    return func(self, *args2, **kwargs2)
  File "/usr/local/lib/python2.7/site-packages/sentry/web/frontend/base.py", line 172, in dispatch
    return self.handle(request, *args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/sentry/web/frontend/group_plugin_action.py", line 25, in handle
    response = plugin.get_view_response(request, group)
  File "/usr/local/lib/python2.7/site-packages/sentry/../sentry_youtrack/plugin.py", line 113, in get_view_response
    return super(YouTrackPlugin, self).get_view_response(request, group)
  File "/usr/local/lib/python2.7/site-packages/sentry/plugins/base/v1.py", line 296, in get_view_response
    response = self.view(request, group)
  File "/usr/local/lib/python2.7/site-packages/sentry/../sentry_youtrack/plugin.py", line 131, in view
    return view(request, group, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/sentry/plugins/bases/issue.py", line 169, in view
    form = self.get_new_issue_form(request, group, event)
  File "/usr/local/lib/python2.7/site-packages/sentry/../sentry_youtrack/plugin.py", line 79, in get_new_issue_form
    initial=self.get_initial_form_data(request, group, event))
  File "/usr/local/lib/python2.7/site-packages/sentry/../sentry_youtrack/forms.py", line 31, in __init__
    self.add_project_fields(project_fields)
  File "/usr/local/lib/python2.7/site-packages/sentry/../sentry_youtrack/forms.py", line 36, in add_project_fields
    form_field = self._get_form_field(field)
  File "/usr/local/lib/python2.7/site-packages/sentry/../sentry_youtrack/forms.py", line 68, in _get_form_field
    'initial': self._get_initial(project_field['name'])}
  File "/usr/local/lib/python2.7/site-packages/sentry/../sentry_youtrack/forms.py", line 57, in _get_initial
    field_key = md5(field_name).hexdigest()
UnicodeEncodeError: 'ascii' codec can't encode character u'\xe4' in position 9: ordinal not in range(128)
```

**Comment by bogdal** [comment]
Fixed in version `0.3.3`.
